opensuse build fails

```
-- HDF5 C compiler wrapper is unable to compile a minimal HDF5 program.
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:[23](https://github.com/espressopp/espressopp/runs/5429930182?check_suite_focus=true#step:6:23)0 (message):
-- Configuring incomplete, errors occurred!
  Could NOT find HDF5 (missing: HDF5_LIBRARIES HDF5_INCLUDE_DIRS
See also "/__w/espressopp/espressopp/build/CMakeFiles/CMakeOutput.log".
  HDF5_HL_LIBRARIES C HL) (found version "")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindHDF5.cmake:1001 (find_package_handle_standard_args)
  CMakeLists.txt:70 (find_package)
```

@XzzX next one:
```
In file included from /__w/espressopp/espressopp/src/interaction/LJcos.cpp:25:
/__w/espressopp/espressopp/src/interaction/VerletListAdressInteractionTemplate.hpp:1091:25: error: 'bind2nd<std::divides<double>, double>' is deprecated: use 'std::bind' instead [-Werror,-Wdeprecated-declarations]
                   std::bind2nd(std::divides<real>(), Volume));
                        ^
/usr/bin/../lib/gcc/x86_64-redhat-linux/12/../../../../include/c++/12/backward/binders.h:170:5: note: 'bind2nd<std::divides<double>, double>' has been explicitly marked deprecated here
    _GLIBCXX11_DEPRECATED_SUGGEST("std::bind")
    ^
```

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]
-        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_rolling', 'ubuntu_mpich']
+        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_rolling']
         CMAKE_COMPILER: [
             '-DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++', 
             '-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++'

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]
-        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_rolling']
+        DISTRO: ['fedora', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_rolling']
         CMAKE_COMPILER: [
             '-DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++', 
             '-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++'

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -7,11 +7,10 @@ on:
 
 jobs:
   build:
-    continue-on-error: ${{ matrix.DISTRO == 'ubuntu_devel' }}
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]
-        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_devel', 'ubuntu_rolling', 'ubuntu_mpich']
+        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_rolling', 'ubuntu_mpich']
         CMAKE_COMPILER: [
             '-DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++', 
             '-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++'

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -7,6 +7,7 @@ on:
 
 jobs:
   build:
+    continue-on-error: ${{ matrix.DISTRO == 'ubuntu_devel' }}
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]

